src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -1157,6 +1157,10 @@ emit_sri_vector (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSignature *fsi
 		} else {
 			return emit_simd_ins_for_sig (cfg, klass, OP_VECTOR_IABS, -1, arg0_type, fsig, args);
 		}
+#elif defined(TARGET_WASM)
+		int opcode = type_enum_is_float(arg0_type) ? OP_XOP_X_X : OP_VECTOR_IABS;
```

Comment:
This looks a little hard to understand, it makes it look like OP_VECTOR_IABS has an intrins argument as well. An if (is_float) would look better.